lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -666,7 +666,7 @@ def test_account_creation_and_enrollment_with_csv_with_blank_lines(self, info_lo
         """
         csv_content = b"\ntest_student@example.com,test_student_1,tester1,USA\n\n"
         uploaded_file = SimpleUploadedFile("temp.csv", csv_content)
-        response = self.client.post(self.url, {'students_list': uploaded_file})
+        response = self.client.post(self.url, {'students_list': uploaded_file, 'email-students': True})
```

Comment:
Flagging that this may be a backwards incompatible api change.  It does make it more consistent with the other batch api methods though.